src/shared/services/bolt/boltConnection.ts

```diff
@@ -50,44 +49,25 @@ export const hasMultiDbSupport = async (): Promise<boolean> => {
 
 export const validateConnection = (
   driver: Driver | null,
-  res: (driver: Driver) => void,
-  rej: (error?: any) => void
+  resolve: (driver: Driver) => void,
+  reject: (error?: any) => void
 ): void => {
   if (driver === null) {
-    rej()
+    reject()
     return
   }
 
+  if (!driver || !driver.session) reject('No connection')
+
   driver
-    .supportsMultiDb()
-    .then((multiDbSupport: boolean) => {
-      if (!driver || !driver.session) return rej('No connection')
-      const session = driver.session({
-        defaultAccessMode: neo4j.session.READ,
-        database: multiDbSupport ? 'system' : undefined
-      })
-      const txFn = buildTxFunctionByMode(session)
-      txFn &&
-        txFn((tx: { run: (query: string) => void }) =>
-          tx.run('CALL db.indexes()')
-        )
-          .then(() => {
-            session.close()
-            res(driver)
-          })
-          .catch((e: { code: string; message: string }) => {
-            session.close()
-            // Only invalidate the connection if not available
-            // or not authed
-            // or credentials have expired
-            if (!e.code || isBoltConnectionErrorCode(e.code)) {
-              rej(e)
-            } else {
-              res(driver)
-            }
-          })
+    .verifyConnectivity()
```

Comment:
Unfortunately it seems that something in the session creation with the database parameter of system is having side effects. With this change running some of the tests results in a "NotSystemDatabase" type of error when running commands when Browser thinks it is using the system database.

I'm not sure why this is happening, but for now I think we need to revert to the old logic, and just switch try switching the query from `CALL db.indexes()` to `CALL db.ping()`

Reply:
Tested printing out `driver.session()`. It seems like in both cases they're the same:
- In `master` branch:
<img width="534" alt="Screenshot 2021-11-09 at 11 47 50" src="https://user-images.githubusercontent.com/26452483/140918912-05bb7fb4-4729-4e10-bcde-8a55e97ef8c9.png">
- In this PR before reverting the changes:
<img width="545" alt="Screenshot 2021-11-09 at 11 50 39" src="https://user-images.githubusercontent.com/26452483/140919216-09834f27-f558-4afa-8c5b-03a110a71aed.png">

This is the problem we have no idea yet. Will probably revisit it in the future.